CHANGES.md

```diff
@@ -2,6 +2,7 @@
 
 ## next
 - Fixed code instrumentation disabling via `tasks.instrumentCode.enabled`
+- Check if used Kotlin version matches the bundled one
```

Comment:
AFAIU it's not only Kotlin version, but also apiCompatibility/stdlib issue. please clarify (see also notes about output messages)

> "...matches the one bundled in target IDE"